frontend/common/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Link from "../Link";
+
+export interface BreadcrumbsProps {
+  links: { title: string; href: string }[];
```

Comment:
@tristan-orourke 
It did not like
`href?: string`
<Link> threw complaints

Funnily enough, looking at the args for Storybook, the third thing passed in has no url but it works just fine in npm run storybook funnily enough

Reply:
nevermind that last bit, storybook args with no href broke
odd but whatever

Reply:
It makes sense that Link would require href to be defined, a link isn't meaningful without that. What you can do if href is optional is conditionally not use a Link at all - I'll show what I mean in another comment.